frontend/src/api/k8s/lmEval.ts

```diff
@@ -17,42 +19,41 @@ export const listModelEvaluations = async (namespace: string): Promise<LMEvalKin
   }).then((listResource) => listResource.items);
 
 const assembleModelEvaluation = (
-  model: string,
-  evalConfig: {
-    batchSize?: string;
-    timeout?: number;
-    taskList: {
-      taskNames: string[];
-    };
-  },
+  data: LmEvalFormData,
   namespace: string,
-  name?: string,
+  batchSize?: string,
 ): LMEvalKind => ({
   apiVersion: kindApiVersion(LMEvalModel),
   kind: LMEvalModel.kind,
   metadata: {
-    name: name || `eval-${translateDisplayNameForK8s(model)}`,
+    name: data.evaluationName || `eval-${translateDisplayNameForK8s(data.model.name)}`,
     namespace,
   },
   spec: {
-    model,
-    ...evalConfig,
+    allowCodeExecution: data.allowRemoteCode,
+    allowOnline: data.allowOnline,
+    ...(batchSize && { batchSize }),
+    logSamples: true,
+    model: data.modelType,
+    modelArgs: convertModelArgs(data.model),
+    taskList: {
+      taskNames: data.tasks,
+    },
+    outputs: {
+      pvcManaged: {
+        size: '5Gi',
```

Comment:
for now change this to `100Mi` so we don't utilize our cluster storage for testing